import old module versions tip (#320)
spacelift-io/user-documentation: docs/vendors/terraform/module-registry.md

## Versions

!!! tip

>     If you would like to import old module versions, you can use a bash script that lists the tags in the repo and then use [spacectl](https://github.com/spacelift-io/spacectl) module create-version --sha xyz  to create versions from them.

Reviewer: Actually, after checking the help for the `spacectl module create-version` command, the command should be:

```suggestion
    If you would like to import old module versions, you can use a bash script that lists the tags in the repo and then use [spacectl](https://github.com/spacelift-io/spacectl){: rel="nofollow"}:
   ```
    spacectl module create-version --id <MODULE ID> --sha <COMMIT SHA> --version <MODULE VERSION>
    ```    
```
